openfecwebapp/api_caller.py

```diff
@@ -148,7 +148,25 @@ def landing_mock_data():
 def load_top_candidates(sort):
         response = _call_api(
             'candidates', 'totals',
-            sort_hide_null=True, election_year=2016, cycle=2016, election_full=True, sort=sort, per_page=5
+            sort_hide_null=True, election_year=2016, cycle=2016, election_full=False, sort=sort, per_page=5
+        )
+        if response['results']:
+            return response['results']
+        return {}
+
+def load_top_pacs(sort):
+        response = _call_api(
+            'totals', 'pac',
+            sort_hide_null=True, cycle=2016, sort=sort, per_page=5
+        )
+        if response['results']:
```

Comment:
I see the other methods are also written this way as well, it might be worth modifying them just to make the code a bit more resilient to potential errors.  I ran into this scenario when I tested the code against a database that didn't have the requisite tables/views in place yet, so the `response` object didn't get filled out with the data that was expected.